tests/decimal_tests.rs

```diff
@@ -3373,10 +3373,18 @@ fn it_can_parse_scientific_notation() {
         ("12345E-28", "0.0000000000000000000000012345"),
         ("1.2345E0", "1.2345"),
         ("1E28", "10000000000000000000000000000"),
+        (
+            "-20165.4676_e-+4294967292",
+            "ERR:Scale exceeds the maximum precision allowed: 4294967292 > 28",
```

Comment:
Minor: Stylistically, I'd probably shift this test case to use a `Result` instead - just to keep "the way to write tests" consistent (e.g. like this: https://github.com/paupino/rust-decimal/blob/46fb4c3c517bc0c27cd534b65f9e8b57c24ba18e/tests/decimal_tests.rs#L3243-L3275)

Reply:
Sorry for the late reply.  Pls go ahead and make this change.  Your style definitely look better.

Feel free to close this PR and redo it your way.

Reply:
Let's merge your PR first so that you get credit for the contribution. We can always modify the style of the test afterwards (to be fair there are a few to make consistent).